database/migrations/2019_09_28_171753_config_fix.php

```diff
@@ -0,0 +1,577 @@
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'sorting_Photos_col',
+				'value' => 'title',
+				'cat' => 'Gallery',
+				'type_range' => 'id|takestamp|title|description|public|star|type',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'sorting_Photos_order',
+				'value' => 'ASC',
+				'cat' => 'Gallery',
+				'type_range' => 'ASC|DESC',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'sorting_Albums_col',
+				'value' => 'max_takestamp',
+				'cat' => 'Gallery',
+				'type_range' => 'id|title|description|public|max_takestamp|min_takestamp',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'sorting_Albums_order',
+				'value' => 'ASC',
+				'cat' => 'Gallery',
+				'type_range' => 'ASC|DESC',
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'imagick',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'dropbox_key',
+				'value' => '',
+				'cat' => 'Admin',
+				'type_range' => STRING,
+				'confidentiality' => '3',
+			],
+			[
+				'key' => 'skip_duplicates',
+				'value' => '0',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'lang',
+				'value' => 'en',
+				'cat' => 'Gallery',
+				'type_range' => DISABLED,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'layout',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => TERNARY,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'image_overlay',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'default_license',
+				'value' => 'none',
+				'cat' => 'Gallery',
+				'type_range' => STRING_REQ,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_max_width',
+				'value' => '0',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_max_height',
+				'value' => '360',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_max_width',
+				'value' => '1920',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_max_height',
+				'value' => '1080',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'full_photo',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'delete_imported',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'Mod_Frame',
+				'value' => '1',
+				'cat' => 'Mod Frame',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'Mod_Frame_refresh',
+				'value' => '30000',
```

Comment:
Another one I meant to mention for a while. I think the unit for this one is completely bonkers. It is milliseconds -- why?! Obviously, it should be seconds.

Reply:
because it goes out of the bat for the frame Javascript code (`timeout`).

Reply:
Oh, I'm aware of JS requirements. I was just pointing out that a user-level config should use a unit that a regular user will expect, and `ms` certainly isn't it. But that's an issue orthogonal to this PR so let's address it separately.